### PR TITLE
[wip] building puppet-agent 1.7.0

### PR DESCRIPTION
**Pull request by komidore64**
NOTE: SRPM_Foundry isn't **quite** ready for prime-time just yet.

The `build.yaml` playbook now assumes that you have RVM installed. I felt it was dirty that the playbook installed it on my system and then immediately filled the default gemset with build gems. So now it looks for an installed ruby, then plops its build gems into a clean gemset.

TODOs:
- [ ] more smoke-testing and confirmation that `SRPM_Foundry` works as expected
- [ ] refactor ansible and patches in a nicer way. could probably accomplish it with ansible's `item` mechanism

### CODE REVIEW

**Comment by ehelms** [comment]
@komidore64 when do you want to merge this?

**Comment by komidore64** [comment]
incoming updates for puppet-agent-1.8.2

File: Rakefile [hunk 1]
```diff
@@ -1,60 +1,52 @@
...
-        $completed_jobs += [j]
-      else
-        puts "#{j} : #{status}"
-      end
-    end
-  end
-end
-
-task :default => 'beaker:machines'
+$completed_jobs = {}
```
Comment (ehelms) [inline Rakefile]: It does not look like there is anything happening in this Rakefile anymore based on the commented code below. Should we just remove it?

Comment (komidore64) [reply Rakefile]: hah yeah i guess that's true! i'll just do that.

Comment (komidore64) [reply Rakefile]: rakefile has been removed

**Review by ehelms — APPROVED** [review]

**Comment by ehelms** [comment]
Thanks @komidore64 !

**Pull request merged**